src/App.scss

```diff
@@ -6,7 +6,7 @@ iframe {
   text-align: center;
 }
 
-.PostInfo {
+/* .PostInfo {
```

Comment:
The CSS styling for the .PostInfo component has been commented out which might affect the display of the posts.